src/main/java/com/sw19/sofa/domain/remind/dto/response/RemindResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.sw19.sofa.domain.remind.dto.response;
+
+import com.sw19.sofa.domain.linkcard.entity.LinkCard;
+import com.sw19.sofa.domain.remind.entity.Remind;
+import com.sw19.sofa.global.util.EncryptionUtil;
+
+import java.time.LocalDateTime;
+
+public record RemindResponse(
+        String encryptedId,
+        String encryptedLinkCardId,
+        String title,
+        String url,
+        String summary,
+        String memo,
+        String imageUrl,
+        long views,
+        LocalDateTime createdAt,
+        LocalDateTime modifiedAt,
+        LocalDateTime visitedAt
+) {
+    public static RemindResponse from(Remind remind) {
+        LinkCard linkCard = remind.getLinkCard();
+        return new RemindResponse(
+                EncryptionUtil.encrypt(remind.getId()),
+                EncryptionUtil.encrypt(linkCard.getId()),
```

Comment:
entitiy 내에 id 를 암호화해주는 getEncryptionId() 메소드를 구현해놓은것으로 알고 있습니다!
해당 메소드가 사용되도록 변경 부탁드립니다
```suggestion
remind.getEncryptId();
 linkCard.getEncryptId();
```

Reply:
entity에 구현된 암호화 메서드를 직접 사용되게 한거 아닌가요

Reply:
똑같은 기능입니다. 다만 해당 역할의 경우 책임이 Entity에 있다고 생각되어 해당 로직을 엔티티 내부 함수 사용으로 변경하면 어떨까 말씀 드린겁니다!

Reply:
수정했습니다.